pkg/scheduler/plugins/gang/gang.go

```diff
@@ -136,13 +136,16 @@ func (gp *gangPlugin) OnSessionOpen(ssn *framework.Session) {
 	ssn.AddJobOrderFn(gp.Name(), jobOrderFn)
 	ssn.AddJobReadyFn(gp.Name(), func(obj interface{}) bool {
 		ji := obj.(*api.JobInfo)
-		return ji.Ready()
+		if ji.CheckTaskMinAvailableReady() && ji.Ready() {
```

Comment:
l think `jobReadyFn` should use another function instead of sharing `CheckTaskMinAvailableReady`, it should not contains `status == Pipelined`

Reply:
Yes, I'll fix it.